dependency.json

```diff
@@ -1 +1 @@
-{"mdl":"data/german.model"}
+{"mdl":"data/senna_embedding.pkl"}
```

Comment:
We have to deal with this differently, hard-coding this into the file under version control seems like a bad idea.